x-pack/legacy/plugins/transform/public/app/sections/transform_management/components/transform_list/action_start.tsx

```diff
@@ -45,55 +41,52 @@ export const StartAction: FC<StartActionProps> = ({ items, forceDisable }) => {
   };
   const openModal = () => setModalVisible(true);
 
-  const buttonStartText = i18n.translate('xpack.ml.dataframe.transformList.startActionName', {
+  const buttonStartText = i18n.translate('xpack.transform.transformList.startActionName', {
     defaultMessage: 'Start',
   });
 
   // Disable start for batch transforms which have completed.
-  const completedBatchTransform = items.some((i: DataFrameTransformListRow) =>
-    isCompletedBatchTransform(i)
-  );
+  const completedBatchTransform = items.some((i: TransformListRow) => isCompletedBatchTransform(i));
   // Disable start action if one of the transforms is already started or trying to restart will throw error
   const startedTransform = items.some(
-    (i: DataFrameTransformListRow) => i.stats.state === DATA_FRAME_TRANSFORM_STATE.STARTED
+    (i: TransformListRow) => i.stats.state === TRANSFORM_STATE.STARTED
   );
 
   let startedTransformMessage;
   let completedBatchTransformMessage;
 
   if (isBulkAction === true) {
     startedTransformMessage = i18n.translate(
-      'xpack.ml.dataframe.transformList.startedTransformBulkToolTip',
+      'xpack.transform.transformList.startedTransformBulkToolTip',
       {
-        defaultMessage: 'One or more selected data frame transforms is already started.',
+        defaultMessage: 'One or more selected transforms is already started.',
       }
     );
     completedBatchTransformMessage = i18n.translate(
-      'xpack.ml.dataframe.transformList.completeBatchTransformBulkActionToolTip',
+      'xpack.transform.transformList.completeBatchTransformBulkActionToolTip',
       {
         defaultMessage:
-          'One or more selected data frame transforms is a completed batch transform and cannot be restarted.',
+          'One or more selected transforms is a completed batch transform and cannot be restarted.',
```

Comment:
Rephrase to `One or more selected transforms are completed batch transforms and cannot be restarted.` Or even drop the word `selected`?

Reply:
Fixed in ecc2c59.